README.rst

```diff
@@ -27,53 +119,73 @@ To install this module you have to install some components:
 - MySQL 5.6.14 or above (http://dev.mysql.com/downloads/mysql/)
 - gcc-c++ and gcc libraries
 
+This module also needs the installation of these other components:
+
+- Fiware-facts module installed (https://github.com/telefonicaid/fiware-facts)
+- A running instance of Orion Context Broker (https://github.com/telefonicaid/fiware-orion)
+- fiware-monitoring connected to the Orion instance to provide
+  information about servers (https://github.com/telefonicaid/fiware-monitoring).
+
+`Top`__.
+
+__ `FIWARE Policy Manager GE - Cloto`_
 
 Installation
-============
+------------
 
 Once you have all prerequisites installed, you must create a DB named cloto in your MySQL server.
-Ensure your mysql path is in your path. If not, you can add executing (change /usr/local/ with your mysql folder):
-    export PATH=$PATH:/usr/local/mysql/bin
+Ensure your mysql path is in your path. If not, you can add executing (change ``/usr/local/`` with your mysql folder):
+
+``export PATH=$PATH:/usr/local/mysql/bin``
```

Comment:
It's a common practice to include the ``$`` prompt (or ``#``, if executed as root) as prefix for shell interactive commands:

    $ export PATH=$PATH:/usr/local/mysql/bin